content/master/concepts/pods.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Crossplane Pods
+weight: 1
+description: Background on the components installed with Crossplane and their functions.
+---
+
+The base Crossplane installation consists of two pods, the `crossplane` pod and
+the `rbac-manager` pod. Both pods install in the `crossplane-system`
+namespace by default. 
+
+
+## Crossplane pod
+
+### Init container
+Before starting the core Crossplane container an _init_ container runs. The init
+container sets parameters related to the core Crossplane container and
+installing the core Crossplane 
+[Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+(`CRDs`).
+
+{{<hint "tip" >}}
+The Kubernetes documentation contains more information about 
+[init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/).
+{{< /hint >}}
+
+The settings the init container sets include installing Provider or Configuration 
+packages with Crossplane, customizing the namespace Crossplane installs in and 
+defining webhook configurations. 
+
+The core CRDs installed by the init container include: 
+* Composite Resource Definitions, Compositions, Configurations and Providers
+* `locks` to manage package dependencies
+* `controllerconfigs` to apply settings to installed Providers
+* `storeconfigs` for connecting external secret stores like 
+[Hashicorp Vault](https://www.vaultproject.io/)
+
+{{< hint "note" >}}
+
+The [Install Crossplane]({{< ref "../software/install" >}}) section has more
+information about customizing the Crossplane install.
+{{< /hint >}}
+
+The status `Init` on the Crossplane pod is the init container running. 
+
+```shell
+kubectl get pods -n crossplane-system
+NAME                                       READY   STATUS     RESTARTS   AGE
+crossplane-9f6d5cd7b-r9j8w                 0/1     Init:0/1   0          6s
+```
+
+The init container completes and starts the Crossplane core container 
+automatically.
+
+```shell
+kubectl get pods -n crossplane-system
+NAME                                       READY   STATUS    RESTARTS   AGE
+crossplane-9f6d5cd7b-r9j8w                 1/1     Running   0          15s
+```
+
+### Core container
+
+The main Crossplane container, called the _core_ container, enforces
+the desired state of Crossplane resources, manages leader elections and process
+webhooks. 
+
+#### Reconcile loop
+
+The core container operates on a _reconcile loop_, constantly checking the 
+status of deployed resources and correcting any "drift." After checking a 
+resource Crossplane waits some time and checks again.
```

Comment:
I don't think this should be moved, since the page flow is about the pods and their interactions/responsibilities but I do think the [root concepts page](https://docs.crossplane.io/v1.12/concepts/) cover this at or near the top instead of being a rehash of what's in the [Introduction](https://docs.crossplane.io/v1.12/getting-started/introduction/) page and what's spelled out in the individual concept sections.

Reply:
Yeah, I agree. It's not so much that I don't want it mentioned here insomuch as I think it should appear somewhere more prominent. If that were the case I could see this being a note on this page rather than an entire heading.

Reply:
I understand. Making it just a note doesn't feel like it's enough. I think the solution is we need to give reconciliation more attention in the parent /concepts page.